pkg/bgpv1/gobgp/server.go

```diff
@@ -95,11 +110,27 @@ func NewGoBGPServer(ctx context.Context, log *logrus.Entry, params types.ServerP
 		return nil, fmt.Errorf("failed starting BGP server: %w", err)
 	}
 
+	if err := s.AddPolicy(ctx, &gobgp.AddPolicyRequest{Policy: allowLocalPolicy}); err != nil {
```

Comment:
Could you add a comment on why this policy & policy assignment are needed ad what they do?